chapters/research-design.tex

```diff
@@ -1,288 +1,586 @@
+
+\section{Causality, inference, and identification}
+
+When we are discussing the types of inputs -- ``treatments'' -- commonly referred to as
+``programs'' or ``interventions'', we are typically attempting to obtain estimates
+of program-specific \textbf{treatment effects}
+These are the changes in outcomes attributable to the treatment.\cite{abadie2018econometric}
+  \index{treatment effect}
+The primary goal of research design is to establish \textbf{causal identification} for an effect.
+Causal identification means establishing that a change in an input directly altered an outcome.
+  \index{identification}
+When a study is well-identified, then we can say with confidence
+that our estimate of the treatment effect would,
+with an infinite amount of data,
+give us a precise estimate of that treatment effect.
+Under this condition, we can proceed to draw evidence from the limited samples we have access to,
+using statistical techniques to express the uncertainty of not having infinite data.
+Without identification, we cannot say that the estimate would be accurate,
+even with unlimited data, and therefore cannot attribute it to the treatment
+in the small samples that we typically have access to.
+Conversely, more data is not a substitute for a well-identified experimental design.
+Therefore it is important to understand how exactly your study
+identifies its estimate of treatment effects,
+so you can calculate and interpret those estimates appropriately.
+All the study designs we discuss here use the \textbf{potential outcomes} framework
+to compare a group that received some treatment to another, counterfactual group.
+Each of these approaches can be used in two contexts:
+\textbf{experimental} designs, in which the research team
+is directly responsible for creating the variation in treatment,
+and \textbf{quasi-experimental} designs, in which the team
+identifies a ``natural'' source of variation and uses it for identification.
+Neither approach is implicitly better or worse,
+and both are capable of achieving effect identification under different contexts.
+
+%-----------------------------------------------------------------------------------------------
+\subsection{Estimating treatment effects using control groups}
+
+The key assumption behind estimating treatment effects is that every
+person, facility, or village (or whatever the unit of intervention is)
+has two possible states: their outcomes if they do not receive some treatment
+and their outcomes if they do receive that treatment.
+Each unit's treatment effect is the individual difference between these two states,
+and the \textbf{average treatment effect (ATE)} is the average of all
+individual differences across the potentially treated population.
+  \index{average treatment effect}
+This is the parameter that most research designs attempt to estimate.
+Their goal is to establish a ``counterfactual scenario'' for the treatment group
```

Comment:
why is counterfactual scenario marked this way instead of bold as the other terms? 

also, add a sidenote and/or index for counterfactual.

Reply:
6876047e5d9ad88115446120d2f7ae3446e81459

Reply:
56941c406c46660c75cfd8a4ca7d3bc3423dae52